src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => (
+
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+    )}
+
+    <p className="Person__partner">
+      {isMarried
+        ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
Don't use nested ternars. Move expression to variable

Reply:
fixed